### PR TITLE
Adds a connection for shows nearby shows

### PR DESCRIPTION
**Pull request by orta**
and some minor TypeScriptification around the place

screen shot 2018-11-06 at 3 30 39 pm

### CODE REVIEW

**Comment by orta** [comment]
I also added descriptions to all of the show fields

File: src/schema/show.ts [hunk 1]
```diff
@@ -386,7 +413,38 @@ export const ShowType = new GraphQLObjectType({
...
+        // OK, looks good
+        const coordinates = show.location.coordinates
+        const gravityOptions = {
+          ...args,
+          displayable: true,
+          near: `${coordinates.lat},${coordinates.lng}`,
+          max_distance: LOCAL_DISCOVERY_RADIUS_KM,
+        }
+
+        return showsLoader(gravityOptions).then(response => {
```
Comment (mzikherman) [inline src/schema/show.ts]: I think `gravityOptions` might be missing the `parseRelayOptions` stuff that will convert from the `pageable` params (`first`, `after`, etc.) into `page` and `size` (to send to gravity).

File: src/schema/show.ts [hunk 2]
```diff
@@ -386,7 +413,38 @@ export const ShowType = new GraphQLObjectType({
       description: "The exhibition title",
       resolve: ({ name }) => (isExisty(name) ? name.trim() : name),
     },
+    nearbyShows: {
+      description: "Shows that are near (~75km) from this show",
+      type: showConnection,
+      args: pageable({
+        size: {
```
Comment (mzikherman) [inline src/schema/show.ts]: Why `size`? As in, a connection and `pageable` already gives you `first` and `last` (for specifying number of items to return).

File: src/schema/show.ts (hunk 1 above)
Comment (orta) [reply src/schema/show.ts]: Great point! Moving this to WIP to add all that - thanks Matt!

File: src/schema/fair.ts [hunk 3]
```diff
@@ -155,8 +113,8 @@ const FairType = new GraphQLObjectType({
       type: Profile.type,
       resolve: (
         { default_profile_id, organizer },
-        options,
-        request,
+        _options,
```
Comment (ashleyjelks) [inline src/schema/fair.ts]: Why prefix with underscores?

Comment (mzikherman) [reply src/schema/fair.ts]: There's sort of a convention (not specific to any language or framework), that variables which are unused get prefixed with an underscore (for reader clarity).

Comment (l2succes) [reply src/schema/fair.ts]: It's a way to silence `jslint` when a variable is unused

File: src/schema/fair.ts [hunk 4]
```diff
@@ -175,14 +133,14 @@ const FairType = new GraphQLObjectType({
...
         },
       }),
       resolve: (
         { id },
         options,
-        request,
+        _request,
         { rootValue: { fairBoothsLoader } }
       ) => {
         const gravityOptions = omit(parseRelayOptions(options), ["page"])
```
Comment (l2succes) [inline src/schema/fair.ts]: 👍

File: src/schema/fair.ts (hunk 3 above)
Comment (orta) [reply src/schema/fair.ts]: Also, there are languages where you can't ignore any params, and the underscore is meant to mean intentionally left blank

**Pull request merged**